.github/workflows/bundles-mac.yml

```diff
@@ -10,11 +10,11 @@ on:
         required: true
         type: string
       java-version:
-        default: '18'
+        default: '19'
```

Comment:
We also need to add a Mac M1 local Runner (possibly by copying this workflow)

Reply:
best to tackle it in a different PR?

Reply:
Sure, indeed